cmd/gateway-gcs-layer.go

```diff
@@ -162,14 +183,14 @@ func newGCSGateway(endpoint string, projectID, secretKey string, secure bool) (G
 		return nil, err
 	}
 
-	anonClient, err := minio.NewCore("storage.googleapis.com", "", "", secure)
+	anonClient, err := minio.NewCore(endpoint, "", "", secure)
```

Comment:
I agree, but this adds a lot of extra code where we will parse S3 XML output. Also we need to maintain the same code in and the minio-go client and here. Minio-go has been added as library already by the s3 gateway.